tests/resources/MST_mirror_2f_measurements.schema.yml

```diff
@@ -39,3 +39,21 @@ data:
         required: false
         units: cm
         type: double
+      - name: mirror_x
```

Comment:
I don't think this is needed for this schema.

Reply:
Related to above comment about derive_mirror_rnda application. (Using this as a mirror_list)

Reply:
I don't understand why the `derive_mirror_rnda` application requires the positions of the mirrors. It simulates them separately to derive the mirror roughness of the mirror panel. It doesn't have anything to do with the dish.